scene/gui/line_edit.cpp

```diff
@@ -619,6 +619,44 @@ void LineEdit::gui_input(const Ref<InputEvent> &p_event) {
 		return;
 	}
 
+	// Open context menu.
+	if (context_menu_enabled) {
+		if (k->is_action("ui_menu", true)) {
+			_update_context_menu();
+			Point2 pos = Point2(get_caret_pixel_pos().x, (get_size().y + theme_cache.font->get_height(theme_cache.font_size)) / 2);
+			menu->set_position(get_screen_position() + pos);
+			menu->reset_size();
+			menu->popup();
+			menu->grab_focus();
+
+			accept_event();
+			return;
+		}
+	}
+
+	if (is_shortcut_keys_enabled()) {
+		// Cut
```

Comment:
That's not cut.
Also the comment is pointless, the `ui_copy` already tells this.

Reply:
Just thought to match the `Cut / Paste` and `Undo / Redo` comments, but I understand that the input action name is self explanatory.